easybuild/easyconfigs/l/librosa/librosa-0.7.2-foss-2019b-Python-3.7.4.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'PythonBundle'
+
+name = 'librosa'
+version = '0.7.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://librosa.github.io'
+description = "Python module for audio and music processing"
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('SciPy-bundle', '2019.10', versionsuffix),
+    ('FFmpeg', '4.2.1'),
+    ('matplotlib', '3.1.1', versionsuffix),
+    ('scikit-learn', '0.21.3', versionsuffix),
+    ('numba', '0.47.0', versionsuffix),
```

Comment:
This should be updated to a newer version of `numba` that supports `LLVM` 9.0.0...

Reply:
Or we add an exception for `librosa` like we already do for other stuff, cfr. 0c49e15